source/v4/channels.yaml

```diff
@@ -49,6 +49,53 @@
           $ref: '#/responses/Unauthorized'
         '403':
           $ref: '#/responses/Forbidden'
+    put:
+      tags:
+        - channels
+      summary: Update a channel
+      description: |
+        Update a channel.
+        ##### Permissions
+        If updating a public channel, `manage_public_channel_members` permission is required. If creating a private channel, `manage_private_channel_members` permission is required.
```

Comment:
Oh, actually `If creating a private channel` needs to be `If updating a private channel`